backend/src/main/java/com/cruru/applicant/domain/repository/EvaluationRepository.java

```diff
@@ -5,10 +5,19 @@
 import com.cruru.process.domain.Process;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.transaction.annotation.Transactional;
 
 public interface EvaluationRepository extends JpaRepository<Evaluation, Long> {
 
     List<Evaluation> findAllByProcessAndApplicant(Process process, Applicant applicant);
 
     void deleteByProcessId(long processId);
+
+    @Modifying
```

Comment:
flush와 clear를 명시해야 1차 캐시에 있는 쓰기 지연 sql이 실행되고 부정합이 생기지 않습니다!

```suggestion
    @Modifying(clearAutomatically = true, flushAutomatically = true)
```

아래 어노테이션들도 수정해주세요!